tck/README.md

```diff
@@ -570,32 +559,29 @@ public class MyIdentityProcessorTest extends IdentityProcessorVerification<Integ
+`Publisher`s and `Subscriber`s with TCK verifications. If a library only implements `Subscriber`s, it does not have to implement `Publisher` tests, the same applies to `IdentityProcessorVerification`-it is not needed if the library does not contain `Processor`s.
 
-In the case of Subscriber Verification are two styles of verifications to available: Blackbox or Whitebox.
+In the case of `Subscriber` Verification are two styles of verifications to available: Blackbox or Whitebox.
 It is *strongly* recommend to test `Subscriber` implementations with the `SubscriberWhiteboxVerification` as it is able to
 verify most of the specification. The `SubscriberBlackboxVerification` should only be used as a fallback,
-once you are sure implementing the whitebox version will not be possible for your implementation - if that happens
-feel free to open a ticket on the [reactive-streams-jvm](https://github.com/reactive-streams/reactive-streams-jvm) project explaining what stopped you from
-implementing the whitebox verification.
+once it's certain that implementing the whitebox version will not be possible—if that happens
+feel free to open a ticket on the [reactive-streams-jvm](https://github.com/reactive-streams/reactive-streams-jvm) project explaining what made implementing the whitebox verification impossible.
 
-In summary: implementations are required to use Verifications for the parts of the specification that they implement,
-and suggest using the whitebox verification over blackbox for the subscriber whenever possible.
+In summary: implementations are required to use Verifications for the parts of the Specification that they implement,
+and encouraged to using the Whitebox Verification over Blackbox for `Subscriber` whenever possible.
 
 ## Upgrading the TCK to newer versions
-While we do not expect the Reactive Streams specification to change in the forseeable future,
-it *may happen* that some semantics may need to change at some point. In this case you should expect test
+While it's not expected for the Reactive Streams Specification to change in the forseeable future,
+it *may be* that some semantics may need to change at some point. In this case it should expected for test
 methods being phased out in terms of deprecation or removal, new tests may also be added over time.
 
-In general this should not be of much concern, unless overriding test methods in your test suite.
-We ask implementers who find the need of overriding provided test methods to reach out via opening tickets
-on the `reactive-streams/reactive-streams-jvm` project, so we can discuss the use case and, most likely, improve the TCK.
+In general this should not be of much concern, unless overriding test methods are overriden by implementers.
+Implementers who find the need of overriding provided test methods are encouraged to reach out via opening Issues
+on the `reactive-streams/reactive-streams-jvm` project, so the use case can be discussed and, most likely, the TCK improved.
```

Comment:
Link to https://github.com/reactive-streams/reactive-streams-jvm instead of repo name?

// I thought I had fixed this hm...